py_ts_interfaces/parser.py

```diff
@@ -87,6 +88,12 @@ def flush(self) -> str:
         self.prepared.clear()
         return "\n\n".join(serialized).strip() + "\n"
 
+    @staticmethod
+    def _get_interface_string(interface: str, should_export: bool):
+        if not should_export or interface[0]=="_":
+            return f"interface {interface}"
+        return f"export interface {interface}"
```

Comment:
We could use some unit tests for this.  For reference, `test_parser_flush` in `tests.py` would be a great template for a new set of parametrized tests that make some assertions that `export` is included.